include/internal/thread_once.h

```diff
@@ -43,3 +51,6 @@
  */
 #define RUN_ONCE(once, init)                                            \
     (CRYPTO_THREAD_run_once(once, init##_ossl_) ? init##_ossl_ret_ : 0)
+
+#define RUN_ONCE_ALT(once, initalt, init)                               \
```

Comment:
Please document, the same way `RUN_ONCE` is documented.  Ideally, all the `DEFINE_RUN_ONCE` and `DECLARE_RUN_ONCE` variants should be similarly documented, but that might be for another PR.

Reply:
Done - I documented all of them.